### ISSUE TITLE
Feature Request

### ISSUE THREAD

**Issue by peteringram0** [mathieudutour/scroll-through-time#7]
Add support for three finger scroll

**Comment by jazzdragon** [comment]
+1
I'm actually using two-finger scrolling to scroll through space, which makes this awesome package unusable for me

**Comment by mallardduck** [comment]
As I was working on #13 I read up on JS detection for such events. Sadly, JS has no native way to distinguish from a true mouse scroll and a trackpad scroll. Because of that there's also limited support for knowing what's going on with 1, 2, vs 3 finger trackpad input.

Maybe some library like Zingtouch could help detect such instances though? Even with that I'm not sure how feasible it would be.

**Comment by jimigwell** [comment]
hi this could be a very great feature.
We can also use cmd+scroll instead of scroll ?
Maybe it is possible to do it via keybinding ? i did not acheive it.

**Comment by sbrl** [comment]
Yep, on a laptop here too. 3 finger scroll would be ideal, but 2 finger + <kbd>CTRL</kbd> would be fine too.

**Comment by mallardduck** [comment]
@sbrl You should be set to use a modifier key already. Just gotta enable the setting.
